tests/ui/lint/invalid_value-polymorphic.rs

```diff
@@ -0,0 +1,13 @@
+// compile-flags: --emit=mir --crate-type=lib -Zmir-opt-level=3
```

Comment:
Hm, don't need `--emit=mir` either.

Reply:
Yeah, if you really want you could use `-Zmir-enable-passes=+InstCombine` and trim the reproducer down all the way to just calling the intrinsic in question, `core::intrinsics::mem_uninitialized_valid::<&T>();`